### PR TITLE
Add -fPIC to allow for dynamic linking

### PR DESCRIPTION
**Pull request by nolden**

### CODE REVIEW

File: libsrc/CMakeLists.txt [hunk 1]
```diff
@@ -49,6 +51,11 @@ add_library(${lib_name} STATIC
   ${ADDITIONAL_SRCS}
   )
 
+CHECK_CXX_COMPILER_FLAG("-fPIC" CXX_HAS_FPIC)
+if(CXX_HAS_FPIC)
+  target_compile_options(${lib_name} PRIVATE "-fPIC")
+endif()
+
```
Comment (jcfr) [inline libsrc/CMakeLists.txt]: Instead, consider using this variable: https://cmake.org/cmake/help/v3.5/variable/CMAKE_POSITION_INDEPENDENT_CODE.html

**Comment by nolden** [comment]
Thanks, didn't know that. Just updated my branch using push --force , is there a better way?

**Comment by jcfr** [comment]
`push --force` is great and well suited for this.

**Comment by fedorov** [comment]
I will merge once TravisCI is green. Thanks!

**Pull request merged**